docs/security/user-certs.md

```diff
@@ -21,9 +21,17 @@ Your **name** and **email** address are encoded into the signed certificate and
 
 ### Requesting a User Certificate
 
-There are two different ways to obtain your certificate, either via a command line interface, or through a web browser. The links below will guide you through whichever method you choose. The command line method is generally simpler and less prone to errors than using a web browser but it may require extra setup and package installation the first time you do it. Most people end up needing their certificate for both command line grid use and web browser use and it is generally easier to import your certificate into your browser (and email client) from the command line certificate files than to export your certificate from your web browser to use on the command line. This is because of the large variety of web browsers each with it's own way to deal with certificates. If you will use the certificate ONLY in your web browser then requesting it from your browser is probably the easiest method.
+You will have to obtain your user certificate using the [CILogon web UI](https://cilogon.org/).
+Follow the steps below to get an user certificate:
+
+1. First, either search for your institution and select it or scroll through list and do the same. <br>![Institution Selection](/img/cilogon_select_idp.png)
+1. When you hit the `Log On` button, you'll be presented by your institution asking for your credentiala.
+1. Upon successfully entering, your credentials, you'll get a page asking for you to enter a password.  You should pick
+and enter a 12 character password and then click on the `Get New Certificate` button.<br>![Password entry](/img/cilogon_cert_password.png) 
```

Comment:
- Does the password have to be exactly 12 characters?
- Active voice: "Enter a 12-character password and click..."

Reply:
>12 chars , clarified